addons/pause/userscript.js

```diff
@@ -15,6 +15,12 @@ export default async function ({ addon, console, msg }) {
   setSrc();
   onPauseChanged(setSrc);
 
+  addon.settings.addEventListener("change", () => {
+    console.log("Settings changed!");
+    if (addon.settings.get("auto-pause") === true) document.addEventListener("focusout", setPaused(false));
```

Comment:
The setting change event is only triggered when  settings change after the addon is already loaded. Also, you probably meant to run `setPaused(true)` here, false will un-pause the project.

Reply:
Oh I misunderstood the method. Thanks!